### PR TITLE
Pino v5

### PR DESCRIPTION
**Pull request by mcollina**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by davidmarkclements** [comment]
@allevo pino 5 has breaking changes – see https://github.com/pinojs/pino/blob/next-major/docs/legacy.md

**Comment by allevo** [comment]
@davidmarkclements thanks! amazing!

**Comment by rodrigooler** [comment]
LGTM

**Comment by delvedor** [comment]
Can we merge this?

File: package.json [hunk 1]
```diff
@@ -112,7 +112,7 @@
     "flatstr": "^1.0.8",
     "light-my-request": "^2.0.3",
     "middie": "^3.1.0",
-    "pino": "^4.17.3",
+    "pino": "^5.0.0-rc.6",
```
Comment (StarpTech) [inline package.json]: pino 5.0.1 is already available

**Comment by mcollina** [comment]
This needs to be updated.

**Comment by mcollina** [comment]
PTAL, this is good to go now.

**Pull request merged**